### PR TITLE
[TECH] Kubernetes Master and Slave Nginx

### PR DESCRIPTION
**Pull request by GuessWhoSamFoo**

### CODE REVIEW

File: docs/applications/containers/linode_kubernetes_guide.md [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+
+    You can now join any number of machines by running the following on each node
+    as root:
+
+      kubeadm join --token 921e92.d4582205da623812 <private IP>:6443 --discovery-token-ca-cert-hash sha256:bd85666b6a97072709b210ddf677245b4d79dab88d61b4a521fc00b0fbcc710c
+    ~~~
+
+    You can create a new token your master host. Enter `kubeadm token create` to generate a new token and pass it as the token parameter in the above command. This token will also only last for 24 hours unless you pass it with the `--ttl infinity` flag
+
+2.  Theres a second set of commands above that which describe how to configure the `kubectl` tool.
```
Comment (EdwardAngert) [inline docs/applications/containers/linode_kubernetes_guide.md]: @GuessWhoSamFoo should this be run from the master or a node?

Comment (GuessWhoSamFoo) [reply docs/applications/containers/linode_kubernetes_guide.md]: Token generation happens on a master node

Comment (EdwardAngert) [reply docs/applications/containers/linode_kubernetes_guide.md]: thanks, @GuessWhoSamFoo 

I'll edit that in and push over the copy edits

**Comment by GuessWhoSamFoo** [comment]
Closed in favor of new PR.

**Pull request closed without merging**